packages/babel-standalone/test/built-into-es5.js

```diff
@@ -30,7 +30,7 @@ describe("@babel/standalone", () => {
     const requireCount =
       babelStandaloneSource.split(/(?<![."])require\(/g).length - 1;
 
-    // 6 vs 13 depends on the build configuration
-    expect([6, 13]).toContain(requireCount);
+    // 8 vs 15 depends on the build configuration
+    expect([8, 15]).toContain(requireCount);
```

Comment:
The new `require(`s are all inside strings, and not actual require calls.

Reply:
I don't know why, but it still has to be the old one locally to pass the test.🤔